datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/MeResolver.java

```diff
@@ -109,7 +110,14 @@ private boolean canManageDomains(final QueryContext context) {
   }
 
   /**
-   * Returns true if the the provided actor is authorized for a particular privilege, false otherwise.
+   * Returns true if the authenticated user has privileges to manage access tokens
+   */
+  private boolean canManageTokens(final QueryContext context) {
```

Comment:
In the file above we have a function defined that does the _exact_ same thing right? 

At least they should have the same name.